doc/user/content/manage/troubleshooting.md

```diff
@@ -10,77 +10,309 @@ aliases:
   - /ops/troubleshooting/
 ---
 
-This page describes several SQL queries you can run to diagnose performance
-issues with Materialize.
+This page describes several SQL queries you can run to diagnose performance issues with Materialize.
 
-{{< warning >}}
-The queries below reference [introspection relations](/sql/system-catalog/mz_internal/#replica-introspection-relations), which are subject to future changes without notice.
+## Mental model and basic terminology
 
-Due to the potential changes in introspection relations all relations below are created in the `mz_internal` schema, and it is not possible to create higher-level views dependent on them.
-{{< /warning >}}
+When you create a materialized view and issue a query, Materialize creates a dataflow. A dataflow consists of instructions on how to respond to data input and to changes to that data. Once executed, the dataflow computes the result of the SQL query, polls the source for updates, and then incrementally updates the query results when new data arrives.
+
+Materialize dataflows act on collections of data. A collection provides a data stream of updates as they happen. To provide fast access to the changes to individual records, the collection can be represented in an alternate form, indexed on data to present the sequence of changes the collection has undergone. This indexed representation is called an arrangement.
+
+
+### Translating SQL to Differential Dataflow
```

Comment:
For now I've removed references to differential dataflow, because it does not add too much value. It may make sense to add it back later (e.g. if we wanted to explain how certain patterns are translated like left joins & certain group by). But right now, I"m not feeling strongly about it.

Reply:
I almost feel we can get really specific in dd details. I don't want to call it out by name so users don't feel they must learn two tools.